### DISCUSSION TITLE
docs: maxPages are missing in the example of useInfiniteQuery docs

### DISCUSSION THREAD

**Discussion by jae-hun-e** [TanStack/query#8152] [General]
First of all, thank you for providing these good tools. This makes it easy to develop.

I am reporting that some of the changes in v5 have not been applied while reading TanningstackQuery/react docs.

'maxPages' added to v5 are not visible in the InfiniteQuery document usage example.
I think this could lead to some people moving on without knowing useful features.
![tmp](https://github.com/user-attachments/assets/cfb30840-093c-4958-b50d-deac3bbf49d8)

If you don't mind, can I fix this problem?
thank you

**Comment by TkDodo** [comment]
Sure 👍
